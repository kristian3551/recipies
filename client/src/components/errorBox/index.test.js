import React from 'react';
import renderer from 'react-test-renderer';
import ErrorBox from './index';

describe('<ErrorBox/>', () => {
    it('should render properly', () => {
        const component = renderer.create(<ErrorBox show={true} errorInfo={'Some error info!'}/>)
        const node = component.toJSON();
        expect(node).toMatchSnapshot();
    });
})