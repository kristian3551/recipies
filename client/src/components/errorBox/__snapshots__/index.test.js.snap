// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorBox/> should render properly 1`] = `
<div
  className="container"
  id="notifications"
  style={
    Object {
      "display": "block",
    }
  }
>
  <div
    class="alert alert-danger"
    id="errorBox"
    role="alert"
  >
    Some error info!
  </div>
</div>
`;
